taichi/codegen/cuda/codegen_cuda.cpp

```diff
@@ -738,11 +738,14 @@ FunctionType CodeGenCUDA::codegen() {
   return CodeGenLLVMCUDA(kernel, ir).gen();
 }
 
+#ifdef TI_WITH_LLVM
```

Comment:
Think this TI_WITH_LLVM macro is redundant, since the current file is located under `taichi/codegen/cuda/`. 

Can you try removing this macro and see if it still compiles?